apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceFeatureExtractors.kt

```diff
@@ -0,0 +1,38 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.logic._logic
+import ch.tutteli.atrium.logic.length
+
+class CharSequenceFeatureExtractors {
```

Comment:
as pointed out, remove this (and the ending `}`

```suggestion
```

We want top-level functions (reformat the code after removal).